script-library/docs/docker-in-docker.md

```diff
@@ -28,14 +28,14 @@
 
 See the [`docker-in-docker`](../../containers/docker-in-docker) definition for a complete working example. However, here are the general steps to use the script:
 
-1. Add [`docker-debian.sh`](../docker-debian.sh) to `.devcontainer/library-scripts`
+1. Add [`docker-debian.sh`](../docker-debian.sh) to `.devcontainer/library-scripts`. Make this file executable: `chmod +x .devcontainer/library-scripts/docker-debian.sh`
 
 2. Add the following to your `.devcontainer/Dockerfile`:
 
     ```Dockerfile
-    COPY library-scripts/*.sh /tmp/library-scripts/
+    COPY .devcontainer/library-scripts/docker-debian.sh /tmp/library-scripts/
```

Comment:
This depends on what "context" is set to in `devcontainer.json`.  If it is `.` you shouldn't add `.devcontainer`.  If is `..`, you should.